package.json

```diff
@@ -39,6 +41,10 @@
     "test:e2e:prod": "npm run servebuild & npm run test:e2e; kill $!"
   },
   "dependencies": {
+    "expo": "~41.0.1",
+    "expo-status-bar": "~1.0.4",
+    "react-native": "https://github.com/expo/react-native/archive/sdk-41.0.0.tar.gz",
```

Comment:
Use the `git+https://git@github.com` protocol, otherwise it will fail the CI due to a bug in npm v7.

Reply:
Setting it using the `git+https://git@github.com` doesn't work it seems to behave differently since we are getting a `tar` rather than a repository. Can you test it out to see if the CI will work with syntax

Reply:
@raineorshine this worked locally.
`"react-native": "git+https://github.com/expo/react-native/archive/sdk-41.0.0.tar.gz"`

Reply:
You can see the checks are failing in this PR. https://github.com/cybersemics/em/pull/1237/checks

Reply:
Fixed